src/components/common/Button/Button.tsx

```diff
@@ -1,13 +1,13 @@
 import styled, { css } from "styled-components";
 
 const shadeColor = (color, percent) => {
-  let R = parseInt(color.substring(1, 3), 16);
-  let G = parseInt(color.substring(3, 5), 16);
-  let B = parseInt(color.substring(5, 7), 16);
+  let R: number = parseInt(color.substring(1, 3), 16);
+  let G: number = parseInt(color.substring(3, 5), 16);
+  let B: number = parseInt(color.substring(5, 7), 16);
```

Comment:
It's fine to explicitly tell typescript that these three variables are only assigned to number type but TypeScript is smart enough to infer that from what you've assigned it to.
For example typescript will assign R,G,B variables to number because you assigned it to parseInt which returns number only.

So with or without assigning a type, typescript knows this!